src/solvers/flattening/bv_endianness_map.cpp

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************\
+
+Module:
+
+Author: Michael Tautschnig
+
+\*******************************************************************/
+
+#include "bv_endianness_map.h"
+
+#include <util/arith_tools.h>
+#include <util/c_types.h>
+
+#include "boolbv_width.h"
+
+void bv_endianness_mapt::build_little_endian(const typet &src)
```

Comment:
Comment either here or in the header file how this differs from `endianness_mapt`?

Reply:
Done - both in the header file and the commit message.